config/config.toml.example

```diff
@@ -369,6 +369,10 @@ store-limit = 0
 # store-liveness-timeout is used to control timeout for store liveness after sending request failed.
 store-liveness-timeout = "5s"
 
+# ttl-based-txn-size decides whether a transaction should update its lock TTL.
+# If the size(in byte) of a transaction is large than `ttl-based-txn-size`, it update the lock TTL during the 2PC.
```

Comment:
should we mention that it only affects the optimistic transaction?

Reply:
It also affect the pessimistic transaction if the pessimistic does not write locks before 2PC.